### DISCUSSION TITLE
Debugger Stack Offset

### DISCUSSION THREAD

**Discussion by ethergrouik** [Vector35/binaryninja-api#2993] [Ideas] [answered]
Hello,
would it be possible for you to add functionality to select the offset reference address?
For example by selecting the address of rbp and that it becomes the offset 0x00.
Thanks in advance

**Comment by psifertex** [comment, marked as answer]
It is is probably a good candidate for a snippet once the new debugger implementation is released. Not something we would necessarily ship by default but very easy to add via a few lines of code in the snippet editor so it can be added to the menu or a hotkey.
